### DISCUSSION TITLE
Run trivy without docker, podman

### DISCUSSION THREAD

**Discussion by andriktr** [aquasecurity/trivy#7531] [Q&A] [answered]
### Question

Hello, 
As far as I understand  things like docker, podman are not necessarily needed to be installed in order to scan images from Azure Container Registry which will be a remote source  and if workload identity will be configured  for container's pod/sa we don't need creds saved in environment library. 
Please correct me if I'm wrong 
Thank you. 

### Target

None

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment, marked as answer]
>As far as I understand things like docker, podman are not necessarily needed to be installed in order to scan images from Azure Container Registry which will be a remote source and if workload identity will be configured for container's pod/sa we don't need creds saved in environment library.

True

**Comment by andriktr** [comment]
@knqyf263 Thanks for clarifying.
